### DISCUSSION TITLE
Do either of YOU want to speak at a Sumit on: The Future of Product Development Summit 2024

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#247] [Ideas]
I was asked to speak about my Skip experience at The Future of Product Development Summit 2024 in Dec.

But I bet the audience would rather hear from one of YOU, makers, builders, founders,  ... interested.

It's a virtual conference style - speakers record a video of themselves giving the talk... demo,  product presentation etc.

**Comment by marcprux** [comment]
That sounds interesting! I'm sure one of us would enjoy speaking. Feel free to contact us (of have them contact us) directly at support@skip.tools with details and we can arrange further.

**Reply by davidakoontz** [reply]
sent an email to support@
